apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/claimFissRead.json

```diff
@@ -1,298 +1,298 @@
 {
```

Comment:
fyi, I checked out this branch and then reverted the changes to the fiss and mcs endpoint responses and then rebuilt the endpoint responses and it did not over write them again so maybe try the same in your branch so that we don't have these extra changes that are not part of the story work.

Reply:
git undo the endpoint responses